packages/bpk-docs/src/components/DocsPageWrapper/DocsPageWrapper.js

```diff
@@ -124,12 +124,7 @@ const DocsPageWrapper = props => {
     if (!platformPreference) {
       return false;
     }
-    return (
-      (platformPreference === 'web' && webSubpage) ||
-      (platformPreference === 'native' && nativeSubpage) ||
-      (platformPreference === 'ios' && iosSubpage) ||
-      (platformPreference === 'android' && androidSubpage)
-    );
+    return platforms[platformPreference];
```

Comment:
Simplified this logic by making use of the existing `platforms` object which holds the subpages.